src/tablestore.rs

```diff
@@ -361,19 +390,22 @@ impl TableStore {
             object_store: self.object_store.clone(),
```

Comment:
Can eliminate `read_blocks_using_index` (as the comment suggests) in a separate PR. Just want to make a note of it.